17-react-native/classworks/Gavruk_Alexander/resources/characters/characters.reducer.js

```diff
@@ -0,0 +1,40 @@
+const initialState = {
+  characters: [],
+  comics: [],
```

Comment:
comics is a separate entity and should be a separate reducer, not a part of characters